doc/_admin-guide/060_Sources/020_File/README.md

```diff
@@ -15,7 +15,7 @@ continues to send messages from this position after the restart.
 
 The file driver has a single required parameter specifying the file to
 open. If you want to use wildcards in the filename, use the [[wildcard-file() source|adm-src-wild]].
```

Comment:
probably this one is solved as well by removing the - from the patterns

Reply:
The full title to match is: "wildcard-file: Collecting messages from multiple text files", hence the usage of the ID.